### ISSUE TITLE
ImageConverter fails with PlatformNotSupported about X509Certificate2UI

### ISSUE THREAD

**Issue by ericstj** [dotnet/corefx#36949]
When using ImageConverter on Linux I see this:
```
System.PlatformNotSupportedException : X509Certificate2UI is not supported on this platform.
```

At the very least we need to fix this exception message.  It'd also be nice if we actually supported these converters on linux: I don't think they rely on Windows functionality.  I think we put them in this assembly because WinForms needed them and we didn't want to put them in TypeConverter and pull Drawing.Common into the shared framework.  @safern did we consider putting these in Drawing.Common?  /cc @Anipik

**Comment by safern** [comment]
This goes back into the long conversation we had when porting these types about pulling `Drawing.Common` down into the shared framework. The main problem here is that we depend on `libgdiplus` for Unix which is an external dependency that we don't distribute as part of the framework.

In order to be able to pull down `Drawing.Common` we would need to either distribute `libgdiplus` ourselves, which doesn't seem feasable to me, or break that dependency, which would take a lot of resources and time... unless I'm missing another option, I don't think we would be able to support this yet.

**Comment by ericstj** [comment]
That's not what I was asking.

Putting the TypeConverters in Drawing.Common doesn't force Drawing.Common into the shared framework, but it does bring TypeConverter into its closure, however TypeConverter is already part of the shared framework and netstandard.

Was that the reason why we didn't put these into Drawing.Common?

**Comment by safern** [comment]
Yes, that was the reason, we were trying to avoid pulling TypeConverter into Drawing.Common's closure.

https://github.com/dotnet/corefx/pull/30092#issuecomment-394451369

**Comment by hughbe** [comment]
I do get the error on some platforms
>  System.PlatformNotSupportedException : X509Certificate2UI is not supported on this platform.
Stack Trace:
   at System.Drawing.ImageConverter..ctor()

No stack trace - kind of confusing!
